hatchways/team-gingerbread
# Stripe API Setup

### What this PR does (required):
- addresses #67 
- sets up Stripe API (keys in Slack)
- routes for creating session (handles transactions by redirecting to Stripe, then redirects back based on payment success/cancelation), creating customer, retrieving customer by id, retrieving all customers, adding a new payment method (e.g., credit card), and retrieving all payment methods for a particular customer

### Any information needed to test this feature (required):
- copy and paste API key from Slack into .env file as STRIPE_API_KEY
- run server
- test routes via Postman (I have already created a few customers, so there should be multiple customers returned via the retrieveAllCustomers route rather than just the ones you create via the createCustomer route)

### Any issues with the current functionality (optional):
- getAllPaymentMethods route shows that the customer does not have any payment methods. need to add this function in
- will have to possibly add or modify routes depending on needs of project
